src/Messenger/MessageHandler/Oilhi/DossierMessageHandler.php

```diff
@@ -28,7 +30,9 @@ public function __invoke(DossierMessage $dossierMessage): void
             service: HookZapierService::TYPE_SERVICE,
             action: HookZapierService::ACTION_PUSH_DOSSIER,
             message: $this->serializer->serialize($dossierMessage, 'json'),
-            response: $response->getContent(throw: false),
+            response: 200 === $response->getStatusCode()
+                ? $response->getContent(throw: false)
+                : $response->getContent(),
```

Comment:
je n'avais pas l'impression que c'était nécessaire puisque la description du paramètre throw est `@param bool $throw Whether an exception should be thrown on 3/4/5xx status codes`

Reply:
ah yes je peux supprimer le paramètre merci